### DISCUSSION TITLE
SSLError(SSLEOFError(8, '[SSL: UNEXPECTED_EOF_WHILE_READING] EOF occurred in violation of protocol (_ssl.c:1000)')

### DISCUSSION THREAD

**Discussion by LokeshNagireddy** [openssl/openssl#24062] [Q&A]
Hi Team,

While we are running our Azure pipelines, on running test cases with robot framework, im getting the below error.

[ WARN ] Retrying (RetryAdapter(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'SSLError(SSLEOFError(8, '[SSL: UNEXPECTED_EOF_WHILE_READING] EOF occurred in violation of protocol (_ssl.c:1000)'))':

This retry is slowing down our process and pipeline taking much time to complete.

I couldnt find any info/document/solution about the reason of this error.

We are using OpenSSL 3.0.2-0ubuntu1.15 from the runner images which is installed on our pipeline agents.

Can some one please help me to find the reason of this error and also any documentation stating the reason of this violations.

i look forward for your response.

**Comment by a0911802160** [comment]
I got the same error
